src/fw_iptables.c

```diff
@@ -484,6 +488,11 @@ iptables_fw_init(void)
 	// CHAIN_TO_ROUTER, packets to HTTP listening on gw_port on router ACCEPT
 	rc |= iptables_do_command("-t filter -A " CHAIN_TO_ROUTER " -p tcp --dport %d -j ACCEPT", gw_port);
 
+	// CHAIN_TO_ROUTER, packets to HTTP listening on fw_port on router ACCEPT
+	fw_port_i = strtol(fw_port, &ptr, 10);
+	rc |= iptables_do_command("-t filter -A " CHAIN_TO_ROUTER " -p tcp --dport %d -j ACCEPT", fw_port_i);
```

Comment:
`if (config->forwarding_enable == 1)` is missing here.

Reply:
Allowing users to router on forwardingport at all times is intended as it not only for access to a forwarded authentication page for clients, but also for access to an admin UI page. It does nothing if not used and saves the need for having adminUiEnable option.
(I need to do a documentation update too! I'll move that up my todo list.)